src/moisture_reading_test.py

```diff
@@ -0,0 +1,27 @@
+import moisture_reading
+import sqlite3
+from datetime import datetime
+
+conn = sqlite3.connect('soil_moisture_test.db')
```

Comment:
Have you tested with the live DB?
Do you need to check that works?

Reply:
Not tested the real DB, should be OK if the test one works but will check